db/migrate/20230728065249_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true, index: true, index: true
+      t.references :post, null: false, foreign_key: true, index: true, index: true
+      t.string :Text
```

Comment:
[REQUIRED] Kindly change the data type of ':Text' to text not string. That is more appropriate than string since It accepts more character.